installer/ClientInstaller.cs

```diff
@@ -171,6 +183,10 @@ class BuildInstaller
 		string version = "";
 		string arch = "";
 		bool isAllLangs = false;
+		Dictionary<string, bool> features = new Dictionary<string, bool>();
+
+		features["g15"] = false;
+		features["overlay"] = false;
```

Comment:
I think there is opportunity to implement this more strongly typed, with named fields/a named tuple instead of using strings (magic keys known at various places), but this is fine too.

Reply:
You mean by creating a dedicated struct/class to pass instead? Yeah that is indeed probably better :+1: